action/protocol/rewarding/protocol.go

```diff
@@ -96,6 +112,8 @@ func (p *Protocol) CreatePreStates(ctx context.Context, sm protocol.StateManager
 		return p.SetReward(ctx, sm, g.DardanellesBlockReward(), true)
 	case g.GreenlandBlockHeight:
 		return p.migrateValueGreenland(ctx, sm)
+	case g.KamchatkaBlockHeight:
+		return p.setFoundationBonusExtension(ctx, sm)
```

Comment:
at Kamchatka height, we add the bonus Start/End epoch, so the business logic now solely depends on `admin{}` stored in statedb